app/config/init.js

```diff
@@ -36,6 +36,10 @@ const _init = function (cfg) {
       notify('Error reading configuration: `config` key is missing');
       return _extractDefault(cfg.defaultCfg);
     }
+
+    // Use options from default config  when not specified in user config #1588
+    _cfg.config = Object.assign({}, _extractDefault(cfg.defaultCfg).config, _cfg.config);
```

Comment:
Just curious. Are we sure the `config` will always be flat or shall it be a deepMerge here instead of `Object.assign`?

Reply:
Thank you for review. I'll change it.